specs/commandline/policy.md

```diff
@@ -0,0 +1,162 @@
+# notation policy
+
+## Description
+
+As part of signature verification workflow, users need to configure the trust policies to specify trusted identities that signed the artifacts, and the level of signature verification to use. For more details, see [trust policy specification and examples](https://github.com/notaryproject/notaryproject/blob/v1.0.0-rc.2/specs/trust-store-trust-policy.md#trust-policy).
+
+The `notation policy` command provides a user-friendly way to manage trust policies. It allows users to import and export trust policies from/to a JSON file. To get started user can use following sample trust policy. In this sample, there are four policies configured for different requirements:
+
+- The Policy named "wabbit-networks-images" is for verifying images signed by Wabbit Networks and stored in two repositories `registry.acme-rockets.io/software/net-monitor` and `registry.acme-rockets.io/software/net-logger`.
+
+- Policy named "unsigned-image" is for skipping the verification on unsigned images stored in repository `registry.acme-rockets.io/software/unsigned/net-utils`.
+- Policy "allow-expired-images" is for logging instead of failing expired images stored in repository `"registry.acme-rockets.io/software/legacy/metrics`.
```

Comment:
nit: typo
```suggestion
- Policy "allow-expired-images" is for logging instead of failing expired images stored in repository `registry.acme-rockets.io/software/legacy/metrics`.
```